docs/core/tools/dotnet-install-script.md

```diff
@@ -183,7 +183,8 @@ You can install a specific version using the `--version` argument. The version m
   Windows:
 
   ```powershell
-  @powershell -NoProfile -ExecutionPolicy unrestricted -Command "[Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12; &([scriptblock]::Create((Invoke-WebRequest -UseBasicParsing 'https://dot.net/v1/dotnet-install.ps1'))) <additional install-script args>"
+  #Run an seperate powershell process because the script calls exit and so will end the current powershell session.
```

Comment:
```suggestion
  # Run a separate PowerShell process because the script calls exit, so it will end the current PowerShell session.
```

Reply:
Thanks for pointing out my poor grammar.

Reply:
No worries! Make sure you click the button Commit suggestion to get them incorporated.